auto-value-gson/src/main/java/com/ryanharter/auto/value/gson/GsonDefaultBuilder.java

```diff
@@ -0,0 +1,16 @@
+package com.ryanharter.auto.value.gson;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * If present, indicates that the annotated method should be used for retrieving an instance of the
+ * AutoValue.Builder. This provides a mechanism for supplying default values.
+ */
+@Retention(SOURCE)
+@Target(METHOD)
+public @interface GsonDefaultBuilder {
```

Comment:
Wondering if this should maybe be called something more like `AutoValueGsonBuilder` to make it specific